ui/utils.py

```diff
@@ -457,3 +460,27 @@ def get_error_response_summary_dict(response):
     else:
         summary_dict = {"content": response.text}
     return {"code": response.status_code, "url": response.url, **summary_dict}
+
+def now_in_utc():
+    """
+    Get the current time in UTC
+
+    Returns:
+        datetime.datetime: A datetime object for the current time
+    """
+    return datetime.datetime.now(tz=pytz.UTC)
+
+
+def send_refresh_request(full_api_url):
+    access_token_url = urljoin(full_api_url, '/oauth2/access_token/')
+    data = {
+        "grant_type": "client_credentials",
+        "client_id": settings.EDXORG_CLIENT_ID,
+        "client_secret": settings.EDXORG_CLIENT_SECRET,
```

Comment:
Maybe for clarity and consistency, we use the same key names as what's been used in [mitxpro](https://github.com/mitodl/mitxpro/blob/master/app.json#L374-L380)